packages/shared/babel-ast-utils/src/index.js

```diff
@@ -35,7 +35,12 @@ export async function parse({
         allowReturnOutsideFunction: true,
         strictMode: false,
         sourceType: 'module',
-        plugins: ['exportDefaultFrom', 'exportNamespaceFrom', 'dynamicImport'],
+        plugins: [
+          'exportDefaultFrom',
+          'exportNamespaceFrom',
+          'dynamicImport',
+          ['moduleAttributes', {version: 'may-2020'}],
```

Comment:
Do we want to enable this by default, or only if a user enables this in their `.babelrc`? At least in that case breaking changes to the spec can be opted-in by apps...

Reply:
This makes sense. It would mean that this functionality requires Babel — which is understandable since this is experimental in the first place.

Should we expose this functionality in some way that doesn't require experimental features?